index.html

```diff
@@ -0,0 +1,154 @@
+            <p class="cards_description">Deep understanding comes from being able to toggle back and forth between focused and diffused thinking.</p>
+          </div>
+          <div class="cards__item"><img src="images/cards-recall.png" alt="Recall" class="cards-image">
+            <h3 class="cards__title">Recall</h3>
+            <p class="cards_description">Improving your ability to remember key bits of information helps to connect different parts of the brain.</p>
+          </div>
+          <div class="cards__item"><img src="images/cards-interliving.png" alt="Interliving" class="cards-image">
+            <h3 class="cards__title">Interliving</h3>
+            <p class="cards_description">When you learn multiple skills at the same time, they reinforce each other.</p>
+          </div>
+          <div class="cards__item"><img src="images/cards-question.png" alt="Questions" class="cards-image">
+            <h3 class="cards__title">Questions</h3>
+            <p class="cards_description">When listening to your teacher, think up a good question, something that you're really itching to know.</p>
+          </div>
+          <div class="cards__item"><img src="images/cards-competence.png" alt="Illusion of Competence" class="cards-image">
+            <h3 class="cards__title">Illusion of Competence</h3>
+            <p class="cards_description">Test yourself, write it down, tell other people, and repeat the process in different settings. Only then can you be truly certain you've understood something.
+            </p>
+          </div>
+        </div>
+      </section>
+      <section class="oakley">
+        <div class="two-columns">
+          <h2 class="two-columns__brief">Barbara Oakley's story.</h2>
+          <div class="two-columns__main-text">
+            <p class="two-columns__paragraph">Math didn't come naturally to Barbara in childhood. She thought of herself as more of a humanities person, and not a brilliant one at that. With an eye to becoming a linguist, after school she enlisted in the army, who sent her to learn Russian. After several years of service, during which she rose to the rank of Captain, Barbara left the army and decided to study engineering. In was during this time that she developed her learning methodology as she was forced to reprogram her brain for a different kind of learning. It turns out that when something doesn't come naturally to you, having to work hard to gain the knowledge that other people may get at first glance gives you a deeper, more comprehensive understanding of the subject matter.</p>
+          </div>
+        </div>
+      </section>
+      <section class="feynman">
+        <h2 class="feynman__title">The Feynman Technique</h2>
+        <p class="feynman__subtitle">Learn without forgetting</p>
+        <a  href="https://fs.blog/feynman-learning-technique/" class="link feynman__link" target="_blank">More &rarr;</a>
+      </section>
+      <section class="kaufman">
```

Comment:
Although project's brief suggested setting up `section__title` and `section__subtitle` as BEM blocks, your approach of setting them as section's block elements is also valid. 

However, according to [BEM Official Documentation](https://en.bem.info/methodology/quick-start/#element) _element is
a composite part of a block that can't be used separately from it_. So we need to ensure that BEM elements are used within their respective blocks to maintain the proper BEM structure and hierarchy, where BEM elements are always contained within their respective blocks.

```
<section class="block">
    <h2 class="block__element">...</h2>
    <p class="block__element">...</p>
</section>
```